salt/utils/cloud.py

```diff
@@ -910,7 +910,12 @@ def connect(self):
         return self._client.connect()
 
     def disconnect(self):
-        self._client.cleanup()  # This removes the lingering PAExec binary
+        try:
+            # This removes any lingering PAExec binaries
+            self._client.cleanup()
+        except CannotDelete as exc:
+            # We shouldn't hard crash here, so just log the error
+            log.debug("Exception cleaning up PAexec: %s", repr(exc))
```

Comment:
```suggestion
            log.debug("Exception cleaning up PAexec: %r", exc)
```
Info you really want a `repr()` of an exception....